### PR TITLE
Refactor Search

### PR DESCRIPTION
**Pull request by runnabro**
- **Remove jQuery dependency.**
- Refactor search popover.
- Fix search popover cutting off last result.
- Add border-radius to sidebar when stacked at smaller sizes.

### CODE REVIEW

**Review by tosih — COMMENTED** [review]
image

Noticed this, can we suppress the search suggestions? `autocomplete="off"`

File: _includes/search.html [hunk 1]
```diff
@@ -1,4 +1,2 @@
 <input placeholder="Search…" type="search" id="search" class="grid-block shrink search" spellcheck="false">
```
Comment (tosih) [inline _includes/search.html]: Can we add `autocomplete="off"`?

**Review by tosih — CHANGES_REQUESTED** [review]

File: _includes/head.html [hunk 2]
```diff
@@ -6,7 +6,6 @@
   <link rel="stylesheet" href="//runnable.com/styles/index.css">
   <link rel="stylesheet" href="{{ site.baseurl }}/css/index.css" type="text/css" media="screen" >
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ site.url }}/feed.xml">
-  <script src="{{ site.baseurl }}/js/jquery.min.js"></script>
```
Comment (taylordolan) [inline _includes/head.html:9]: You should remove the file too

**Review by taylordolan — COMMENTED** [review]
^

**Review by taylordolan — APPROVED** [review]

**Review by tosih — APPROVED** [review]

**Pull request merged**